src/components/Capture/Face.js

```diff
@@ -41,16 +41,24 @@ class Face extends Component {
   }
 
   handleCapture = (payload) => {
-    const { actions, nextStep, mobileFlow } = this.props
+    const { actions, mobileFlow } = this.props
     const id = randomId()
     const faceCaptureData = {
       ...defaultPayload,
       ...payload,
       sdkMetadata: addDeviceRelatedProperties(payload.sdkMetadata, mobileFlow),
       id,
     }
-    actions.createCapture(faceCaptureData)
-    nextStep()
+
+    if (!this.props.captures.face) {
+      actions.createCapture(faceCaptureData)
+    }
+  }
+
+  componentDidUpdate() {
+    if (this.props.captures.face) {
```

Comment:
Shouldn't there be a check that the face capture is different from the previous face capture (if any) also?

Reply:
We only want to create the face capture once (when the button is clicked) and we only want to go to the next step once the face capture has been created. I hope that makes sense